16/alpine/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM hmctspublic.azurecr.io/imported/library/node:16-alpine
```

Comment:
we could probably simplify all this given they are the same and just set an arg for the tag? instead of 12,14,16 folders just have an alpine and debian-slim folder what do you think?

Reply:
Agreed! Much better. Implemented as suggested - CI successfully run here: https://github.com/hmcts/cnp-node-base/actions#:~:text=Actor-,Added%20GitHub%20Actions%20Workflows.,-ci%20%2313%3A%20Pull